src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+:root {
+  --icon-big: url(/src/images/Search.svg) 26px 50% no-repeat;
+  --icon-small: url(/src/images/Search.svg) 13px 50% no-repeat;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: inherit;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.big {
+  height: 70px;
+  background: var(--icon-big);
+  padding-left: 62px;
```

Comment:
Instead of having big and small input, you can make one of them the default one, and add modifier to another